### PR TITLE
Update updateApis to use raml-toolkit

### PR DESCRIPTION
**Pull request by wjhsf**
Changed updateApis command to use raml-toolkit and removed unnecessary code.
`npm run updateApis` should have identical functionality to before this PR, except I changed the names of some folders.

Note: This passes tests, but `npm run updateApis` will fail until the next release of raml-toolkit.

### CODE REVIEW

File: .depcheckrc.yml [hunk 1]
```diff
@@ -1,16 +1,6 @@
...
-    'nyc',
-    'prettier',
-    'tslib',
-    'typescript',
-    'typedoc-plugin-external-module-map',
-    'typedoc-plugin-nojekyll',
-    'utility-types',
-  ]
+  - gulp-typescript
+  # Unused and could be removed, but leaving because it could be helpful
```
Comment (kalagarraj) [inline .depcheckrc.yml:3]: helpful for what usecase?

File: package.json [hunk 2]
```diff
@@ -29,7 +29,10 @@
     "doc:resources": "cp CHANGELOG.md ./docs",
     "clean": "rm -rf renderedTemplates dist",
     "renderTemplates": "gulp --gulpfile ./gulpfiles/renderTemplatesTask.ts renderTemplates",
-    "updateApis": "gulp --gulpfile ./gulpfiles/downloadRamlsTask.ts updateApis",
+    "backupApis": "rm -rf updateApisTmp && mkdir updateApisTmp && mv apis updateApisTmp/oldApis",
```
Comment (kalagarraj) [inline package.json:32]: prefer backup dir name as "updateApisBackup". Users exactly what this dir is for..

File: testIntegration/tests/authHelper.test.js [hunk 3]
```diff
@@ -26,15 +26,15 @@ describe("JS: Auth Helper getShopperToken", () => {
         {
           authType: "guest",
           customerId: "ab1avTqmPIwc4MRq0cEs59LEka",
-          preferredLocale: "en_US"
+          preferredLocale: "en_US",
```
Comment (kalagarraj) [inline testIntegration/tests/authHelper.test.js:29]: why "," ?

File: testIntegration/tests/authHelper.test.js [hunk 4]
```diff
@@ -26,15 +26,15 @@ describe("JS: Auth Helper getShopperToken", () => {
...
         },
         { Authorization: "Bearer AUTH_TOKEN" }
       );
 
     const shopperToken = await helpers.getShopperToken(
       {
         baseUri: "http://somewhere",
-        parameters: { organizationId: "foo" }
+        parameters: { organizationId: "foo" },
```
Comment (kalagarraj) [inline testIntegration/tests/authHelper.test.js:37]: typo ","  ?

File: testIntegration/tests/authHelper.test.js (hunk 3 above)
Comment (wjhsf) [reply testIntegration/tests/authHelper.test.js:29]: A recent update to eslint changed our comma dangle rule. They're now required.

File: .depcheckrc.yml (hunk 1 above)
Comment (wjhsf) [reply .depcheckrc.yml:3]: Helpful for creating type definitions. We don't currently do anything complex with our types, but we definitely could.

**Comment by monkey-jeff** [comment]
@wjhsf can we get these conflicts resolved and we will get this merged today?

File: package.json (hunk 2 above)
Comment (wjhsf) [reply package.json:32]: This directory isn't just for the backups, it's for all of the files generated by `updateApis`. The backups are stored in a directory within this directory.

**Pull request merged**